api/tests/unit/domain/models/PixEduBadgeAcquisitionOrderer_test.js

```diff
@@ -59,19 +59,19 @@ describe('Unit | Domain | Models | PixEduBadgeAcquisitionOrderer', function () {
 
           // then
           expect(highestBadge).to.deep.equal(
-            domainBuilder.buildBadgeAcquisition.forPixEduFormationContinue1erDegreMaitre()
+            domainBuilder.buildBadgeAcquisition.forPixEduFormationContinue2ndDegreExpert()
```

Comment:
Cela ne devrait il pas etre `formateur`?

> should return the PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_FORMATEUR badge acquisition

Reply:
En fait c'est le it qui devrait etre

should return the PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_EXPERT badge acquisition

Reply:
Corrigé